src/app/View.ts

```diff
@@ -71,33 +67,31 @@ export class View {
     playersChosenName: string,
     playersChosenAvatarPath: string,
   ): void {
-    const newPlayer = new Player(
-      playersChosenName,
-      playersChosenAvatarPath,
-    );
-
-    const newTimer = new Timer(15);
     const playerName = Render.elementFactory(
       'h3',
       {
         className: 'player__name',
       },
       playersChosenName,
     );
-    //create elements
     const playerAvatar = Render.elementFactory('img', {
       className: 'player__avatar',
       src: playersChosenAvatarPath,
     });
-    const remainingTime = Render.elementFactory(
-      'div',
-      {
-        className: 'remainig-time__counter',
-      },
-
-      `Time Left : ${newTimer.theTurnTimeLeft} s`,
+    const gameController = new GameController(this);
+    gameController.initializePlayer(
+      playersChosenName,
+      playersChosenAvatarPath,
     );
-
+    const remainingTime = Render.elementFactory('div', {
+      id: 'time-remaining',
+      className: 'remainig-time__counter',
+    });
+    const herdView = Render.elementFactory('div');
+    const rollResultView = Render.elementFactory('div');
+    rollResultView.addEventListener('click', () => {
```

Comment:
consider moving all listeners to the bottom of the file, they seem to be lost within UI parts.